examples/hosts/app-integration/external-data/src/mock-service-interface/dataParsing.ts

```diff
@@ -12,6 +12,12 @@ export interface ParsedTaskData {
     priority: number;
 }
 
+interface TaskData {
```

Comment:
We could probably put this shared definition somewhere under `model-interface` - all of the sub-directories should be able to import those common definitions. Then we wouldn't have to define this in multiple places.